addons/procurement/models/procurement.py

```diff
@@ -290,11 +290,13 @@ def run_scheduler(self, use_new_cursor=False, company_id=False):
 
             # Check done procurements
             procurements = ProcurementSudo.search([('state', '=', 'running')] + (company_id and [('company_id', '=', company_id)] or []))
+            checked_procurements = list()
```

Comment:
why not [] ?

Reply:
no particular reason

Reply:
please change it to `[]`, it's faster than `list()` and it's more pythonic

Reply:
please, feel free to do it